exp/controlleridentitycreator/awscontrolleridentity_controller.go

```diff
@@ -48,7 +48,7 @@ type AWSControllerIdentityReconciler struct {
 	WatchFilterValue string
 }
 
-// +kubebuilder:rbac:groups=infrastructure.cluster.x-k8s.io,resources=awsclustercontrolleridentities,verbs=get;list;watch
+// +kubebuilder:rbac:groups=infrastructure.cluster.x-k8s.io,resources=awsclustercontrolleridentities,verbs=create,get;list;watch
```

Comment:
Stupid question. Shouldn't this update role.yaml or something? :D

Reply:
good point, I forgot make generate 😓

Reply:
I literally added a github check for that in another project because I constantly forget to run make manifests && make generate. :D

Reply:
Surprisingly, verify job didn't fail 🤔

Reply:
Added.